src/pages/index.tsx

```diff
@@ -160,6 +171,8 @@ const Home: NextPage<Props> = ({
   reasonSearch,
   ptiurn,
   dateRange,
```

Comment:
Could we pass down the dateRange here?

Reply:
Agreed to leave as is to reduce complexity.